src/api/config/cache.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+import (
+	log "github.com/Sirupsen/logrus"
+	"github.com/albrow/zoom"
+)
+
+// Pool is a pool of Zoom connections used by other packages
+var Pool *zoom.Pool
```

Comment:
Thanks @mmikulicic, this is definitely the part I was most concerned about with this PR so thanks for the feedback! I agree that `InitPool` is a better name, and I will move to using an accessor with a mutex.